pay-api/migrations/versions/2024_11_19_8bd139bbb602_.py

```diff
@@ -0,0 +1,26 @@
+"""24454 - EFT File column clean up
+
+Revision ID: 8bd139bbb602
+Revises: 474917a13bd4
+Create Date: 2024-11-19 21:20:17.080264
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+revision = '8bd139bbb602'
+down_revision = '474917a13bd4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table('eft_files', schema=None) as batch_op:
+        batch_op.drop_column('total_deposit_cents')
+        batch_op.drop_column('number_of_details')
```

Comment:
Removed so there is no confusion. These are not accurate as they are extracted directly from the TDI17 which include extra transactions that are not EFT.

We filter by location_id during processing to only get EFT transactions, which the total may not match the file total.